### ISSUE TITLE
[ask] Set area print and worng cell and how sum format in cell

### ISSUE THREAD

**Issue by atmanegara** [aVadim483/fast-excel-templator#3]
In my Excel template I use the print area set, and it doesn't work when outputting a new file and there are also several records that have the wrong cell format as in the example in the following image

![image](https://github.com/aVadim483/fast-excel-templator/assets/9096711/6904ffdb-7909-43e7-a204-f91d0c0228ad)

<img width="450" alt="image" src="https://github.com/aVadim483/fast-excel-templator/assets/9096711/0767e363-b4fa-44e8-bb8c-bf2241ab0f0e">

and how to sum formula with dynamic cells?

[template](https://www.dropbox.com/scl/fi/bxogb3ddknh3qqzsmon7n/buku_kas.xlsx?rlkey=kcwtzxwa20h26pfhnh9l7wz98&dl=0)

source code
https://pastecode.io/s/zv4r09be

**Comment by Raman117** [comment]
how are you rendering the formulas?
i tried to add data in the template but the formulas aren't running before the file is downloaded.

**Comment by aVadim483** [comment]
Yes, the library writes the output file row by row and does not store the values of the written cells in memory, so it cannot calculate the values of formulas. Excel recalculates the values when open the file
